infra.bs

```diff
@@ -803,9 +803,14 @@ to as "U+0029 RIGHT PARENTHESIS", because even though it renders, this avoids un
 <p><a>Code points</a> are sometimes referred to as <a>characters</a> and in certain contexts are
 prefixed with "0x" rather than "U+".
 
-<p>A <dfn export>surrogate</dfn> is a <a>code point</a> that is in the range U+D800 to U+DFFF,
+<p>A <dfn export>lead surrogate</dfn> is a <a>code point</a> that is in the range U+D800 to U+DBFF,
```

Comment:
I think this should be "leading" or "high": https://www.unicode.org/glossary/#leading_surrogate

Reply:
Interesting, the macros in ICU use "lead" and "trail".